test/image/compare_pixels_test.js

```diff
@@ -101,6 +101,7 @@ if(allMock || argv.filter) {
 }
 
 var FLAKY_LIST = [
+    'treemap_coffee',
```

Comment:
cc @archmoj something is off with this new mock. See https://circleci.com/gh/plotly/plotly.js/64411?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link - it failed 5 times in a row.

Reply:
You can pick https://github.com/plotly/plotly.js/commit/0175108c7670910ad2175367767381397ccd3779 to fix that.